TranslatorReasonerAPI.yaml

```diff
@@ -784,10 +796,69 @@ components:
           items:
             $ref: '#/components/schemas/Attribute'
           nullable: true
+        qualifiers:
+          type: array
+          description: >-
+            A list of qualifiers that together provide additional detailed
+            nuance to the assertion beyond what the subject-predicate-object
+            can provide.
+          items:
+            $ref: '#/components/schemas/Qualifier'
+          nullable: true
       additionalProperties: false
       required:
         - subject
         - object
+    Qualifier:
+      additionalProperties: false
+      description: >-
+        An additional nuance attached to an assertion
+      type: object
+      properties:
+        qualifier_type_id:
+          type: string
+          description: >-
+            The category of the qualifier, drawn from a hierarchy of qualifier
+            slots in the Biolink model (e.g. subject_aspect, subject_direction,
+            object_aspect, object_direction, etc).
+          example: subject_aspect
+          nullable: false
+        value:
+          type: string
+          description: >-
+            The value associated with the type of the qualifier, drawn from
+            a set of controlled values by the type as specified in
+            the Biolink model (e.g. 'decreased' or 'increased' for the type
+            subject_direction, etc).
+          example: decreased
+          nullable: false
+      required:
+        - type
+        - value
+    QualifierSet:
+      additionalProperties: false
+      description: >-
+        A set of Qualifiers that act together to add nuance to an assertion.
+      properties:
+        qualifiers:
+          type: array
+          description: >-
+            A list of qualifiers that together provide additional detailed
+            nuance to the assertion beyond what the subject-predicate-object
+            can provide.
+          items:
+            $ref: '#/components/schemas/Qualifier'
+          nullable: false
+        qualified_predicate:
```

Comment:
'qualified_predicate' is the name of an edge property defined in Biolink.  May be confusing to have a TRAPI property with the same name. Consider 'constrained_predicate'?

Reply:
I am neutral, but fine to change if it seems less conflicting with Biolink.

Reply:
I understand better now and after pondering further, I'm convinced that constrained_predicate is better. I have updated the PR.